# Enforce using ellipsis (…) instead of three dots (i18n-ellipsis)

Translatable strings should use ellipsis (…) instead of three dots.

## Rule details

Examples of **incorrect** code for this rule:

```js
__('Continue...');

```

Examples of **correct** code for this rule:

```js
__('Continue…');
```
